File: javascript-closure-angular-client/API/Client/PurchaseHeaderOutPayment.js
goog.provide('API.Client.PurchaseHeaderOut_payment');

/**
 * @record
 */
API.Client.PurchaseHeaderOutPayment = function() {}

/**
 * @type {!API.Client.PaymentTerms}
 * @export
 */
API.Client.PurchaseHeaderOutPayment.prototype.terms;

/**
 * @type {!API.Client.WithholdingMode}
 * @export
 */
API.Client.PurchaseHeaderOutPayment.prototype.withholdingMode;

/**
 * installments
 * @type {!Array<!API.Client.InstallmentComplete>}
 * @export
 */
API.Client.PurchaseHeaderOutPayment.prototype.installments;

File: async-scala-client/src/main/scala/io/swagger/client/model/PurchaseHeaderOut_payment.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class PurchaseHeaderOut_payment (
  terms: Option[PaymentTerms],
withholdingMode: Option[WithholdingMode],
installments: Option[List[InstallmentComplete]]  // installments
)
